tests/migrations/test_operations.py

```diff
@@ -3003,6 +3003,30 @@ def test_run_sql_noop(self):
             operation.database_forwards("test_runsql", editor, None, None)
             operation.database_backwards("test_runsql", editor, None, None)
 
+    def test_run_sql_adds_missing_semicolon_on_collect_sql(self):
+        project_state = self.set_up_test_model('test_runsql')
+        new_state = project_state.clone()
+        operation_with_semicolon = migrations.RunSQL(
+            '''
+            INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1);
+            ''',
```

Comment:
I guess, as we're relying on there being whitespace, we should make this explicit:
```suggestion
            'INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1);\n',
```

Reply:
Good point, thanks!